epsagon/wrappers/chalice.py

```diff
@@ -0,0 +1,31 @@
+"""
+Wrapper for AWS Lambda in Chalice environment.
+"""
+
+from __future__ import absolute_import
+from epsagon.wrappers import lambda_wrapper
+
+
+class ChaliceWrapper:
```

Comment:
Why not using inheritance?
More operators may be implemented, which are currently not wrapped (`__getitem__`, comparison, etc.).
Also, `isinstance` will keep working

Reply:
👍